packages/params/src/forkName.ts

```diff
@@ -110,6 +110,14 @@ export function isForkPostFulu(fork: ForkName): fork is ForkPostFulu {
   return isForkPostElectra(fork) && fork !== ForkName.electra;
 }
 
+export type ForkPreBlobs = ForkPreDeneb;
+export type ForkPostBlobs = ForkPostFulu;
+export type ForkBlobs = ForkName.deneb | ForkName.electra;
+export const forkBlobs = [ForkName.deneb, ForkName.electra];
+export function isForkBlobs(fork: ForkName): fork is ForkBlobs {
+  return fork === ForkName.deneb || fork === ForkName.electra;
+}
+
```

Comment:
remove, per resolution to discussion in https://github.com/ChainSafe/lodestar/pull/7774

Reply:
updated in #8125